docs/ipdk-dpdk.md

```diff
@@ -0,0 +1,123 @@
+# Building IPDK networking recipe for DPDK target
+
+## Overview
+
+This document describes how to build and run IPDK networking recipe on DPDK target.
+
+## Setup
+
+### Build P4 SDE for DPDK
+```bash
+git clone --recursive git@github.com:p4lang/p4-dpdk-target.git p4sde
+```
+
+For build instructions, refer [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-target/blob/main/README.md#building-and-installing)
+
+### Build and install infrap4d dependencies
+
+```bash
+git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
+cd ipdk.recipe
+export IPDK_RECIPE=`pwd`
+cd $IPDK_RECIPE/setup
+cmake . -DCMAKE_INSTALL_PREFIX=`path for installing dependencies`
+make
+```
+
+*Note*: If running as non-sudo user, provide `USE_SUDO = ON` option to cmake config and also use `sudo` for ldconfig
```

Comment:
"running as a non-**root** user"

"provide the `-DUSE_SUDO=ON` option to CMake" (no spaces, and don't forget the `-D`).